game/missiongenerator/tgogenerator.py

```diff
@@ -844,38 +845,53 @@ def create_helipad(
         else:
             self.helipads.append(sg)
 
-        warehouse = Airport(
-            pad.position,
-            self.m.terrain,
-        ).dict()
-        warehouse["coalition"] = "blue" if self.cp.coalition.player else "red"
-        # configure dynamic spawn + hot start of DS, plus dynamic cargo?
-        self.m.warehouses.warehouses[pad.id] = warehouse
```

Comment:
Branch needs to be rebased, otherwise we risk losing a lot of stuff due to git's regular merge "filling in the gaps" incorrectly

Reply:
Actually nvm, it seems to have gotten merged into the right spot...